unlock-app/src/unlock.ts

```diff
@@ -1,12 +1,14 @@
 // This file contains type definitions for the various kinds of data that we use
 // throughout unlock-app.
 
+/* eslint-disable no-unused-vars */
```

Comment:
Maybe a quick comment above this saying that this is a bug in the eslint parser? (with ideally a link to the issue) and a TODO to remove when it's fixed?